lte/gateway/deploy/roles/trfserver/files/traffic_server.py

```diff
@@ -583,12 +594,26 @@ def _setup_iperf3(self):
         for instance in self._instances:
             if instance.is_uplink:
                 iperf = iperf3.Server()
-                iperf.bind_address = '192.168.129.42'
+                ip_str = ipaddress.ip_address(instance.ip)
+                if ip_str.version == 4:
+                  print("Running ipv4")
+                  iperf.bind_address = '192.168.129.42'
+                else:
+                  print("Running ipv6")
+                  iperf.bind_address = '3001::2'
+                  os.system('sudo /sbin/ip -6 route add 3001::10/64 dev eth3')
```

Comment:
It would be nice to define variables to improve readability of this code.

Reply:
done